packages/output/src/electron-node/console-logger-server.ts

```diff
@@ -0,0 +1,83 @@
+/********************************************************************************
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable, postConstruct } from 'inversify';
+import { LoggerWatcher } from '@theia/core/lib/common/logger-watcher';
+import { LogLevelCliContribution } from '@theia/core/lib/node/logger-cli-contribution';
+import { ILoggerServer, ILoggerClient, ConsoleLogger } from '@theia/core/lib/common/logger-protocol';
+import { ChannelServiceImpl, LogOutputChannel } from '../node/channel-server';
+
+@injectable()
+export class ConsoleLoggerServer implements ILoggerServer {
+
+    protected client: ILoggerClient | undefined = undefined;
+
+    @inject(LoggerWatcher)
+    protected watcher: LoggerWatcher;
+
+    @inject(LogLevelCliContribution)
+    protected cli: LogLevelCliContribution;
+
+    @inject(ChannelServiceImpl)
+    protected loggerService: ChannelServiceImpl;
+
+    protected outputChannel: LogOutputChannel;
+
+    @postConstruct()
+    protected init() {
+        for (const name of Object.keys(this.cli.logLevels)) {
+            this.setLogLevel(name, this.cli.logLevels[name]);
+        }
+
+        this.outputChannel = this.loggerService.getChannel('Log (Theia)', 'log');
```

Comment:
not sure about hardcoding `Theia`, could we get rid of it somehow?

Reply:
This code would likely to be rebound in applications that have log files configured, so this could be considered 'default' behavior.  Perhaps I will read the application name and use that.

Reply:
Or simply something generic like `IDE Backend`, since it comes from there?
Maybe `Self`, I don't know.

Reply:
@kittaakos It's backend code.

Reply:
OK, thanks for the note. Please ignore my comment.